docs/guides/compose.md

```diff
@@ -15,7 +15,7 @@ keywords:
 
 <!-- DOCS-SITE-LINK:START automatically generated  -->
 
-### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/guides/compose/)
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/guides/compose)
 
```

Comment:
@czubocha In this document we need to add a section referencing (and linking to) the new shared state buckets. That section should call out out a few things:
- How the shared state helps with collaboration (i.e. outputs are always in sync if different people deploy different services).
- Previous version of compose used local state, that has been deprecated/replaced
- Why commands like `outputs` and `refresh-outputs` are no longer needed (and that they've been deprecated)

Reply:
Right, that should be helpful. I’ve added a new section.